roles/network/tasks/install.yml

```diff
@@ -52,10 +55,32 @@
     - { name: 'net.ipv4.ip_forward', value: '1' }  # Masquerading LAN->Internet
     - { name: 'net.ipv4.conf.default.rp_filter', value: '1' }
     - { name: 'net.ipv4.conf.default.accept_source_route', value: '0' }
-    #- { name: 'kernel.sysrq', value: '1' }             # OS values differ, Ok?
-    - { name: 'kernel.core_uses_pid', value: '1' }
```

Comment:
Ansible output `'kernel.core_uses_pid: 1' +` should be removed ~8 lines above.

Reply:
You can fix it, I'm packing to leave for the weekend